src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -28,10 +29,24 @@ const SkillsQuizStepper = () => {
   };
 
   const { refinementsFromQueryParams } = useContext(SearchContext);
+  const history = useHistory();
+  const location = useLocation();
+  const handleSeeMoreButtonClick = () => {
+    // TODO: incorporate handling of skills related to jobs as well
+    const queryString = new URLSearchParams({ skill_names: refinementsFromQueryParams.skill_names });
+    let path = location.pathname;
+    // remove current page section from url
+    path = path.replace('/skills-quiz', '');
+    path = refinementsFromQueryParams.skill_names ? `${path}/search?${queryString}` : `${path}/search`;
+    path = path.replace(/\/\/+/g, '/'); // to remove duplicate slashes that can occur because of trailing slash url before redirection
```

Comment:
For better understanding, can we write it like this? 

```javascript
let BASE_PATH = location.pathname.replace('/skills-quiz', '');
let SEARCH_PATH = refinementsFromQueryParams.skill_names ? `${BASE_PATH}/search?${queryString}` : `${BASE_PATH}/search`;
SEARCH_PATH = SEARCH_PATH.replace(/\/\/+/g, '/'); // to remove duplicate slashes that can occur because of trailing slash url before redirection
```

Also, we are already destructuring `skill_names` from `refinementsFromQueryParams` [here](https://github.com/edx/frontend-app-learner-portal-enterprise/blob/master/src/components/skills-quiz/SkillsQuizStepper.jsx#L31). Maybe we can make use of that in place of refinementsFromQueryParams.skill_names on line # 36 and 40.

Reply:
I have refactored with good naming convention and about the second tip, that destructed var will be removed with jobs var, its temporary until jobs data isn't available.

Reply:
@sameenfatima78 please have a look now.